pkg/codegen/utilities_types.go

```diff
@@ -230,3 +230,50 @@ func isProvideDefaultsFuncRequiredHelper(t schema.Type, seen map[string]bool) bo
 	}
 	return false
 }
+
+// PackageReferences returns a list of package names that are referenced by the given package.
```

Comment:
This should probably return package references, not just names.

Reply:
Done ✅